### PR TITLE
Joomla 3.10 bot message

### PR DESCRIPTION
**Pull request by zero-24**
As mentiond here: https://github.com/joomla/joomla-cms/discussions/38560#discussioncomment-3454185

This here is the change from 2.5.x (which does not exists) to the 3.10-dev.

@brianteeman I can you please double check the text It should be clear right? Do you think we should link to the magazin article to explain things?

### CODE REVIEW

File: cli/Application/Command/Update/Pulls.php [hunk 1]
```diff
@@ -81,11 +81,11 @@ protected function closePulls()
 		// Only process for joomla/joomla-cms
 		if ($this->project->gh_user == 'joomla' && $this->project->gh_project == 'joomla-cms')
 		{
-			$message = 'Joomla! 2.5 is no longer supported.  Pull requests for this branch are no longer accepted.';
+			$message = 'Thank you for your pull request. Joomla! 3 is in security only mode and does no longer accept pull requests please target your PR against Joomla! 4.';
```
Comment (brianteeman) [inline cli/Application/Command/Update/Pulls.php:84]:
```suggestion
			$message = 'Thank you for your pull request. Joomla! 3 is in security only mode and no longer accepts pull requests. Please target your PR against Joomla! 4.';
```

**Comment by brianteeman** [comment]
I forget - does this get applied to existing PR or just new ones.

If its just new ones then its good to go

**Comment by zero-24** [comment]
From my understanding this runs when updating the database which would include existing PRs. So we first should make sure the open 3.10-dev PRs should be cleaned up. Should not be much left at that end.

**Comment by brianteeman** [comment]
actually there is only one open pr.
this is only for pulls isnt it? not issues

**Comment by zero-24** [comment]
Yes its only PRs.

**Comment by brianteeman** [comment]
all good to go then

**Pull request merged**